app/containers/artist/_stories/_header.story.js

```diff
@@ -0,0 +1,29 @@
+import React from "react"
+import Relay from "react-relay"
+import { storiesOf } from "@kadira/storybook"
+import Header from "../components/header"
+import { ArtistQueryConfig } from "../../../relay/root_queries"
+import artsyNetworkLayer from "../../../relay/config"
+
+import StubContainer from "react-storybooks-relay-container"
+
+storiesOf("Artist Header", module)
+  .add("Real Artist - Leda Catunda", () => {
+    Relay.injectNetworkLayer(artsyNetworkLayer())
```

Comment:
on the long run this will need to be done for all stories, they support [middleware like stuff](https://getstorybook.io/docs/react-storybook/addons/writing-addons) or we can run a function somewhere